testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -44,8 +45,24 @@ pub struct ClusterSwarmKube {
 
 impl ClusterSwarmKube {
     pub async fn new() -> Result<Self> {
-        let result = Config::infer().await;
-        let config = result.map_err(|e| format_err!("Failed to load config: {:?}", e))?;
+        // This uses kubectl proxy locally to forward connections to kubernetes api server
+        Command::new("/usr/local/bin/kubectl")
+            .arg("proxy")
+            .spawn()
+            .expect("Failed to launch kubectl proxy");
```

Comment:
Thanks. Maybe last bit is to return error here(since function anyway returns result), instead of panic?

@bors-libra delegate+

Reply:
Done